src/App.css

```diff
@@ -36,3 +36,171 @@
     transform: rotate(360deg);
   }
 }
+
+.wrapper {
+  max-width: 1500px;
+  margin: 0 auto;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+ul {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+h1 {
+  color: crimson;
+  font-family: 'Helvetica';
+  font-size: 40px;
+  font-weight: bold;
+}
+
+header img {
+  display: block;
+  max-width: 50%;
+  height: auto;
+  margin-top: 35px;
+}
+
+header {
+  display: flex;
+  justify-content: center;
+}
+header a {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+header hr {
+  border: 1px solid black;
+  width: 100vw;
+  margin-bottom: 40px;
+}
+
+.enterPage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+}
+
+.enterPage a {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.enterButton {
+  font-size: 50px;
+  padding: 8px 15px;
+  margin-top: 7px;
+  background: crimson;
+  color: white;
+  border: solid 2px crimson;
+  display: inline-block;
+}
+
+.catalogue {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: center;
+}
+
+.catalogue a img {
+  max-width: 100%;
+  height: 300px;
+  margin: 5px;
+  transition: all 0.2s ease-in-out;
+}
+
+.cardContainer {
+  display: flex;
+  justify-content: space-evenly;
+  align-items: center;
+}
+
+.cardInfoContainer {
+  max-width: 40%;
+}
+
+.cardDisplay img {
+  max-width: 100%;
+  height: 325px;
+  padding: 25px;
+  transition: 0.2s ease-in-out;
+}
+
+.cardDisplay img:hover,
+.cardDisplay img:focus {
+  scale: 1.5;
+}
+
+.catalogue a img:hover,
+.catalogue a img:focus {
+  scale: 1.05;
+  box-shadow: 0px 0px 10px 3px crimson;
+}
+
+.searchCards-button {
+  padding: 8px 15px;
+  margin-top: 7px;
+  background: crimson;
+  color: white;
+  border: solid 2px crimson;
+  display: inline-block;
+}
+
+
+.cardInfoContainer p {
+  margin: 5px 0;
+}
+
+@media (max-width: 1200px) {
+  header img {
+    display: block;
+    max-width: 60%;
+    height: auto;
+  }
+}
+
+@media (max-width: 900px) {
+  header img {
+    display: block;
+    max-width: 70%;
+    height: 175px;
+  }
+}
+
+@media (max-width: 640px) {
+    header img {
+    display: block;
+    max-width: 75%;
+    height: 125px;
+  }
+  .cardContainer {
+  display: flex;
+  flex-direction: column-reverse;
+  align-items: center;
+  margin-bottom: 50px;
+  }
+    .cardDisplay img:hover,
+    .cardDisplay img:focus {
+    scale: 1.25;
+  }
+  .cardInfoContainer {
+    max-width: 65%;
+  }
+}
+
+@media (max-width: 460px) {
+    header img {
+    display: block;
+    max-width: 80%;
+    height: auto;
+  }
+}
```

Comment:
Great use of media queries fully tested and working!

Reply:
Thank you thank you.